rmw_connext_cpp/src/connext_static_publisher_info.cpp

```diff
@@ -56,6 +57,28 @@ rmw_ret_t ConnextStaticPublisherInfo::get_status(
         rmw_offered_deadline_missed->total_count_change =
           offered_deadline_missed.total_count_change;
 
+        break;
+      }
+    case DDS::StatusKind::DDS_OFFERED_INCOMPATIBLE_QOS_STATUS:
+      {
+        DDS::OfferedIncompatibleQosStatus offered_incompatible_qos;
+        DDS::ReturnCode_t dds_return_code =
+          topic_writer_->get_offered_incompatible_qos_status(offered_incompatible_qos);
+
+        rmw_ret_t from_dds = check_dds_ret_code(dds_return_code);
+        std::cout << "from_dds: " << from_dds << std::endl;
+        if (from_dds != RMW_RET_OK) {
```

Comment:
nit: should be `RMW_RET_OK != from_dds` according to our style guide.

Reply:
Done.